assets/src/data/content/writers/html.tsx

```diff
@@ -124,8 +124,12 @@ export class HtmlParser implements WriterImpl {
     );
   }
 
-  p(context: WriterContext, next: Next, _x: Paragraph) {
-    return <p>{next()}</p>;
+  p(context: WriterContext, next: Next, attrs: Paragraph) {
+    return (
+      <p id={'E' + attrs.id} data-annotate="text">
```

Comment:
We will need to go in and add ID and data-annotate attributes throughout where appropriate. 
Note: ID's can't start with a number, prepended "E" here.